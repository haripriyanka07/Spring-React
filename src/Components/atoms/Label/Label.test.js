import React from 'react';
import Label from './Label';
import renderer from 'react-test-renderer';

it('test the Label', () => {
    const asFragment = renderer.create(<Label>New Label</Label>).toJSON();
    expect(asFragment).toMatchSnapshot()
})

