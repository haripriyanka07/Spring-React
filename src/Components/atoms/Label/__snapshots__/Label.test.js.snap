// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the Label 1`] = `
<label>
  New Label
</label>
`;
